### PR TITLE
Improving coredump number check to prevent too many err msgs in log

### PR DESCRIPTION
**Pull request by bentsi**

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -733,15 +733,11 @@ def _get_n_coredumps(self):
...
-        try:
-            n_backtraces_cmd = 'sudo coredumpctl --no-pager --no-legend 2>/dev/null'
-            result = self.remoter.run(n_backtraces_cmd,
-                                      verbose=False, ignore_status=True)
-            return len(result.stdout.splitlines())
-        except Exception as details:
-            self.log.error('Error retrieving number of core dumps : %s',
-                           details)
-            return None
+        n_backtraces_cmd = 'sudo coredumpctl --no-pager --no-legend 2>&1 ||true'
```
Comment (amoskong) [inline sdcm/cluster.py]: We have ignore_status=True in run(), why do we still need the `|| true`?

**Comment by fruch** [comment]
after a second thought (and hearing someone read outloud the [python zen](
https://www.python.org/dev/peps/pep-0020/) in a podcast)

> Errors should never pass silently.
> Unless explicitly silenced.

I think `verbose=False, ignore_status=True` kind of mean that we want errors silenced explicitly.

and since
> There should be one-- and preferably only one --obvious way to do it.

changing the command seems a bit Dutch here :)

**Comment by bentsi** [comment]
@fruch current logic of RemoteFabric is that when error happened we print it any way (whether it was silenced or not)

**Comment by bentsi** [comment]
will merge this, but opened a task in Trello to review verbosity and all commands that use `verbose=False`

File: sdcm/cluster.py (hunk 1 above)
Comment (bentsi) [reply sdcm/cluster.py]: `ignore_status=True` means don't throw exception, this PR is about not to printing error message when command failed (currently if the command fails we print error message any way, whether verbose argument was True or False)

**Pull request merged**